Mlem/Views/Tabs/Posts/Components/Post/Components/Instance and Community List/Accounts Page.swift

```diff
@@ -48,6 +50,7 @@ struct AccountsPage: View {
                 } label: {
                     Image(systemName: "plus")
                 }
+                .accessibilityLabel("Add a new account.")
```

Comment:
Mostly a question but should accessibility labels include punctuation?

Reply:
They can control how Siri pronounced it. Granted I'm not 100% consistent here.